prod-devel-rcar-s4.yaml

```diff
@@ -79,6 +79,7 @@ common_data:
 
     # Generate ext4 image files
     - [IMAGE_FSTYPES:append, " ext4"]
+    - [IMAGE_INSTALL:append, " perf"]
```

Comment:
IIRC, we are doing this by image recipe, not via YAML file.

Reply:
As I see, we still add the required packages by yaml file:
https://github.com/xen-troops/meta-xt-prod-devel-rcar-gen4/blob/master/prod-devel-rcar-s4.yaml#L150
Or should it should be moved to some other file?

Reply:
Ah, okay. Let it be here.